R/dt_formats.R

```diff
@@ -12,15 +12,11 @@ dt_formats_init <- function(data) {
   dt_formats_set(data = data, formats = list())
 }
 
-dt_formats_add <- function(data, formats, prepend) {
```

Comment:
What happens with user code that depends on `prepend`?

Reply:
It never worked anyway. This was a vexing bug but after unraveling it we now have something stable and better tested.

Reply:
Ok, sounds good!